Makefile

```diff
@@ -13,11 +13,9 @@ WITH_NIGHTLY_FEATURES = 1
 endif
 
 TOOLCHAIN_ARGS =
-ifeq ($(shell (rustc --version | grep -q nightly); echo $$?), 0)
```

Comment:
I don't understand the bug description. nightly features are enabled with `--all-features`, there's no need to force using the latest nightly (with `+nightly`) when the local default toolchain is already nightly.

And I don't understand your change either. The `ifeq` is paired with `else` at L21 and `endif` at L25.